single_level.py

```diff
@@ -16,35 +16,36 @@
 
 spreadsheet = os.path.join(config.get("Destinations", "directory"), config.get("Destinations", "filename"))
 
-#DACS Required Notes to check for
-required_notes = ["scopecontent", "accessrestrict"]
+def get_note_contents(resource, array, note_type):
+	notes = resource["notes"]
+	content_list = []
+	for note in notes:
+		try:
+			if note["type"] == note_type:
+				if note["jsonmodel_type"] == "note_singlepart":
+					content_list.append(note["content"].encode('utf-8'))
+				else:
+					content_list.append(note["subnotes"][0]["content"].encode('utf-8'))
+		except:
+			pass
+	return ", ".join(content_list)
```

Comment:
I think we should probably use a more unique character to join these, like maybe a | (vertical bar)